### ISSUE TITLE
[Feature Request] : Add yarn static server package to host docs during yarn startup

### ISSUE THREAD

**Issue by muddlebee** [paritytech/substrate-api-sidecar#923]
**Proposed Change or Idea**

Host https://github.com/paritytech/substrate-api-sidecar/tree/master/docs/dist 
using https://yarnpkg.com/package/static-server during yarn startup

@TarikGul is this a viable feature??

**Comment by muddlebee** [comment]
hey @TarikGul 

I was trying to run the endpoints from the swagger file generated(index.html) at dist/ 
However this is the endpoint initially formed

> 
> curl -X 'GET' \
>   'file://wsl/accounts/3123/balance-info' \
>   -H 'accept: application/json'

I checked the swagger docs, it recommends 
> Open index.html in your HTML editor and replace with the URL for your OpenAPI 3.0 spec.
https://swagger.io/docs/open-source-tools/swagger-ui/usage/installation/
Some steps are mentioned here on how to host.

Found a different plugin https://github.com/scottie1984/swagger-ui-express as well which supports this.

> write our own static server using the native http module. Especially if it's just for serving static files it can be put together pretty simply.

Shall we go with the above approach you suggested? or any alternative let me know, happy to tackle this.
